Aptoide/aptoide-client-v8
# Refactor/asv 1115 remove old app view fragment

**What does this PR do?**

Removes anything related to the old app view that is not needed anymore
Renames NewAppViewFragment to AppViewFragment

**Database changed?**

   No

**How should this be manually tested?**

Try to break AppView to see if anything was removed that wasn’t supposed, specially different entry points.

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-1115

**Code Review Checklist**
